deluge/ui/gtk3/dialogs.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, header, text, icon, buttons, parent=None):
         super(BaseDialog, self).__init__(
             title=header,
             parent=parent if parent else component.get('MainWindow').window,
-            flags=Gtk.DialogFlags.MODAL | Gtk.DialogFlags.DESTROY_WITH_PARENT,
+            flags=Gtk.DialogFlags.MODAL | Gtk.DialogFlags.DESTROY_WITH_PARENT | Gtk.DialogFlags.USE_HEADER_BAR,
```

Comment:
This is not adding anything back and it's a change I do not want to make. I don't agree that using header bar for buttons is a good ui design.